.github/workflows/check.yml

```diff
@@ -94,8 +94,8 @@ jobs:
           cargo +stable check --manifest-path utils/wasm-builder/test-program/Cargo.toml
           cargo +stable check --manifest-path utils/cargo-gbuild/test-program/Cargo.toml --workspace --target wasm32-unknown-unknown
 
-      - name: "Check: crates-io packages"
-        run: cargo +stable run --release -p crates-io check
+      - name: "Check: crates-io packages publishing"
+        run: cargo +stable run --release -p crates-io publish --simulate --registry-path /tmp/cargo-http-registry
```

Comment:
This can also be run locally to simulate publishing. If `--registry-path` was passed, then on error it is possible to continue from the last crate that failed to publish.

Reply:
@clearloop can we also remove `crates-io check` in favor of running `cargo publish` (which does `cargo build` as well)?

Reply:
sure, `crates-io check` is a simple version of the simulation, we df can remove it since we now have a better solution